server/controller/user-contoller.js

```diff
@@ -49,23 +51,23 @@ const logIn = async(req,res,next) => {
     const {email , password} = req.body;
```

Comment:
Consider disabling autocomplete for the password field to prevent the browser from storing or suggesting passwords, which could enhance security.

```suggestion
const {email , password} = req.body; // autocomplete="off"
```